docs/custom-audio.md

```diff
@@ -0,0 +1,15 @@
+# Custom Audio
+To replace a sequences you should only create a zip file with textures to replace at the same path as mk64.o2r. Only wav, mp3, ogg and flac are supported.
```

Comment:
Place a zip file containing audio next to the games executable. Only wav, mp3, ogg and flac are supported.
Or perhaps
Place a zip file containing audio in the same path as mk64.o2r. Only wav, mp3, ogg and flac are supported.